.github/renovate.json

```diff
@@ -7,7 +7,6 @@
 	"schedule": [ "before 3am on the first day of the month" ],
 	"updateNotScheduled": false,
 	"gitIgnoredAuthors": [ "matticbot@users.noreply.github.com" ],
-	"ignoreDeps": [ "mockery/mockery", "php-mock/php-mock", "phpunit/phpunit" ],
```

Comment:
Do we want PHPUnit updates from Renovate, given our current setup that's dependent on different versions?

Reply:
There won't be any, nothing depends on PHPUnit directly anymore. Same for the other two packages here.

Looks like they all got removed in #17954, everything got switched to get PHPUnit via yoast/phpunit-polyfills and to use brain/monkey for mocking.